uiowa/uiowa
# Upated grad programs view to not trim the length of homepage URL.

Resolves #2957 

# How to test

* `blt ds --site=grad.uiowa.edu`
* `drush @grad.local uli academics/manual/academic-program/section-viii-advanced-programs-offered-graduate-college`
* Check links for "Journalism" and "Sustainable Water Development" to ensure they are not truncated.
